liquid-integration/src/androidTest/java/com/e/liquid_integration/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,24 @@
+package com.e.liquid_integration
```

Comment:
This example file can be removed. Unless you're going to implement Instrumented Tests.